### ISSUE TITLE
Respawn time on rare creatures

### ISSUE THREAD

**Issue by pangolp** [chromiecraft/chromiecraft#1884]
### What client do you play on?

esES

### Faction

- [X] Alliance
- [X] Horde

### Content Phase:

- [ ] Generic
- [ ] 1-19
- [ ] 20-29
- [ ] 30-39
- [X] 40-49
- [ ] 50-59

### Current Behaviour

When the server is restarted, most of the rare world creatures reappear. This can affect, in the gleaning of certain items, which are unique to them.

### Expected Blizzlike Behaviour

When restarting the server, either the respawn times established in the database should be respected, or the server restarts should be changed a little less frequently.

### Steps to reproduce the problem

1. Kill some rare creature
2. Restart the server.
3. The creature will reappear.

### Extra Notes

None

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/b2f08934bcc60491f8fa8971f98837b2d48a489e

### Operating system

Ubuntu 20.04

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chromie-xp](https://github.com/azerothcore/mod-chromie-xp)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna-lua-engine](https://github.com/azerothcore/mod-eluna-lua-engine)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-server-auto-shutdown](https://github.com/azerothcore/mod-server-auto-shutdown)
- [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-custom-worldboss](https://github.com/55Honey/Acore_CustomWorldboss)
- [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)

### Customizations

None

### Server

ChromieCraft

**Comment by syssneck** [comment]
Hey, I can't confirm it on my local AC. Rare creatures do not respawn with a server restart. I'm also not sure, what the screenshot has to do with this report?

**Comment by VFaleiro** [comment]
> I'm also not sure, what the screenshot has to do with this report

I think he also meant items sold in limited quantity, which in fact do replenish when server resets. But I'm not sure that should be changed though.

**Comment by VFaleiro** [comment]
@syssneck 
> I'm also not sure, what the screenshot has to do with this report

I think he also meant items sold in limited quantity, which in fact do replenish when server resets (I know because that's how I bought [this cooking recipe](https://wowgaming.altervista.org/aowow/?item=16767)).
I'm not sure that should be changed though.

**Comment by pangolp** [comment]
> Hey, I can't confirm it on my local AC. Rare creatures do not respawn with a server restart. I'm also not sure, what the screenshot has to do with this report?
> Also, ChromieCraft (usually) only restarts once per day, which is not enough, to abuse any drops from rare creatures, even if this was an issue.

Yes, the image does not match the report very well, it is true. In fact, I was actually talking to some people on discord, and I used the topic to show them how reports should be made. The image is because this item has a cooldown of several hours (I do not remember if it was 15) and if the restart occurs, the item reappears in the seller.

**Comment by pangolp** [comment]
> @syssneck
> 
> > I'm also not sure, what the screenshot has to do with this report
> 
> I think he also meant items sold in limited quantity, which in fact do replenish when server resets (I know because that's how I bought [this cooking recipe](https://wowgaming.altervista.org/aowow/?item=16767)). I'm not sure that should be changed though.

Just create the issue, so that we can discuss it. I don't know if my reasoning is correct or not, but that's what we are here for, to detect if it is a bug, and if it is, then report it so that it can be repaired.
